### PR TITLE
Add option to hide "Forgot Password"

### PR DESCRIPTION
**Pull request by emilroz**
This PR adds an option to remove "Forgot password" from the login screen. Feature requested for the LDAP/AD based installations.

After setting:
```
omero config set omero.web.show_forgot_password false
```

The login page should not render "Forgot password" link anymore:

image

With the public user enabled:

image

### CODE REVIEW

**Comment by pwalczysko** [comment]
This is a very interesting option also for our production server at SLS, cc @sbesson - the users are attempting the "Forgot pwd" button even as it does not bring anything

**Comment by sbesson** [comment]
Definitely applicable in our case as this is a no-op. Although in the case of the institutional UoD SLS OMERO, I almost wonder whether Forgot password should redirect you to https://lsd.lifesci.dundee.ac.uk/authentication/reset...

**Comment by pwalczysko** [comment]
> Definitely applicable in our case as this is a no-op. Although in the case of the institutional UoD SLS OMERO, I almost wonder whether Forgot password should redirect you to https://lsd.lifesci.dundee.ac.uk/authentication/reset...

Great idea, I am all for it. This would reduce the frequency of users asking for help on the mailing list by a good 90 percent.

File: omeroweb/settings.py [hunk 1]
```diff
@@ -830,6 +830,15 @@ def leave_none_unset_int(s):
             "(OME team by default)."
         ),
     ],
+    "omero.web.show_forgot_password": [
+        "SHOW_FORGOT_PASSWORD",
+        "true",
+        json.loads,
```
Comment (will-moore) [inline omeroweb/settings.py:836]: This should be `parse_boolean` to be consistent with the other boolean settings.
Although `json.loads("false")` returns `False`, `json.loads("False")` or `json.loads("True")` fail.

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-python-superbuild-push#798](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/798/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/798/consoleText) for more details.
Possible conflicts:
  - PR #270 will-moore 'constrain plate grid'
    - omeroweb/settings.py
  - PR #297 knabar 'Add configuration settings for default search group and user'
    - omeroweb/settings.py
  - PR #303 will-moore 'Timezone from client'
    - omeroweb/settings.py
  - PR #311 emilroz 'Add omero.web config option to set custom favicon'
    - omeroweb/settings.py

--conflicts

File: omeroweb/settings.py (hunk 1 above)
Comment (emilroz) [reply omeroweb/settings.py]: Thanks @will-moore, should be fixed now.

Comment (will-moore) [reply omeroweb/settings.py]: 👍

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-python-superbuild-push#806](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/806/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/806/consoleText) for more details.
Possible conflicts:
  - PR #270 will-moore 'constrain plate grid'
    - omeroweb/settings.py
  - PR #303 will-moore 'Timezone from client'
    - omeroweb/settings.py
  - PR #311 emilroz 'Add omero.web config option to set custom favicon'
    - omeroweb/settings.py
  - Upstream changes
    - omeroweb/settings.py

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-python-superbuild-push#807](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/807/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/807/consoleText) for more details.
Possible conflicts:
  - PR #311 emilroz 'Add omero.web config option to set custom favicon'
    - omeroweb/settings.py
  - Upstream changes
    - omeroweb/settings.py

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-python-superbuild-push#808](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/808/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/808/consoleText) for more details.
Possible conflicts:
  - PR #311 emilroz 'Add omero.web config option to set custom favicon'
    - omeroweb/settings.py
  - Upstream changes
    - omeroweb/settings.py

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-python-superbuild-push#809](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/809/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/809/consoleText) for more details.
Possible conflicts:
  - PR #311 emilroz 'Add omero.web config option to set custom favicon'
    - omeroweb/settings.py
  - Upstream changes
    - omeroweb/settings.py

--conflicts

**Comment by will-moore** [comment]
This is working fine for me locally. Code 👍 . Good to merge.

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-python-superbuild-push#815](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/815/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/815/consoleText) for more details.
Possible conflicts:
  - Upstream changes
    - omeroweb/settings.py

--conflicts

**Comment by chris-allan** [comment]
@emilroz: This needs `master` merged in and conflicts resolved should be able to be merged afterwards.

/cc @sbesson

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-python-superbuild-push#870](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/870/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/870/consoleText) for more details.
Possible conflicts:
  - Upstream changes
    - omeroweb/settings.py

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-python-superbuild-push#871](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/871/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/871/consoleText) for more details.
Possible conflicts:
  - Upstream changes
    - omeroweb/settings.py

--conflicts

**Review by will-moore — APPROVED** [review]
Tested locally after latest merge commit. Looks good.

**Pull request merged**